src/main/java/uk/ac/cam/cl/dtg/segue/etl/ContentIndexer.java

```diff
@@ -848,6 +860,27 @@ private void recordContentTypeSpecificError(final String sha, final Content cont
                     + ") has both children and a value.");
         }
 
+        // Make sure no children of potentially expandable content are expandable, if so record a content error
+        if (((null != content.getLayout() && content.getLayout().equals("tabs")) || content instanceof CodeSnippet) && null != content.getChildren()) {
```

Comment:
Not an issue specific to this PR, but I don't think CodeSnippet is expected to have children in any case.

Reply:
Yep that's true, but including it doesn't hurt I guess? I think I'll leave it for now